layouts/partials/header.html

```diff
@@ -1,11 +1,16 @@
 <header class="header">
 	<div class="container">
-		<div class="logo" role="banner">
-			<a class="logo__link" href="{{ "" | relLangURL }}" title="{{ .Site.Title }}" rel="home">
+		<a class="logo" href="{{ relLangURL "" }}" title="{{ .Site.Title }}" rel="home" role="banner">
```

Comment:
You can optimize CSS a little bit by adding the `.flex` class to` .logo`, like `logo flex`